app/graphql/types/search_event_type.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Types
+  class SearchEventType < Types::BaseObject
```

Comment:
Thought for the future: I'm waffly (and thus not requesting a change) as to whether it might be good to include the `Term.phrase` as part of this externally facing object. I know from the data perspective, it is a relation but from the API perspective we may want this to feel like a single object. Hmmm.

Reply:
I had the same thought and, being similarly ambivalent, stuck to what is contained in the `SearchEvent` model for now. My expectation is that this type will eventually include a lot of data that it does not currently, including the search phrase. (As you noted, the field name will probably also change to reflect this.) For now, sticking with a basic implementation feels to me like a good approach, given that we'll be extending it in the hopefully near future.